operations/resourceaccesstokens.md

```diff
@@ -2,7 +2,7 @@
 
 ## Get all resource access tokens
 
-Returns all resource access tokens based on resource access tokens, reservations or interval. One of them must be specified in the request.
+Returns all resource access tokens based on resource access tokens, reservations or interval. One of them must be specified in the request. Note this operation uses [Pagination](../guidelines/pagination.md).
```

Comment:
```suggestion
Returns all resource access tokens based on resource access tokens identifiers, reservations or interval. One of them must be specified in the request. Note this operation uses [Pagination](../guidelines/pagination.md).
```

Reply:
Yes I agree that's better, but I would drop the 's' on access tokens and make it "based on resource access token identifiers"